index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"> 
+    <title>Abel's Amazing Work</title>
+</head>
+<body>
+    <header id="header">
+        <nav>
+            <h3 id="welcome">Welcome</h3>
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon-md" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </nav>
+    </header>
```

Comment:
- [ ] It was a b=good idea to use the `<header>` tag to wrap the content of the header, but the `<h3>` is supposed to be a link `<a>` and should be outside of the `<nav>` tag since it's actually a logo and we need to click on it.
- [ ] For the hamburger icon(SVG) you need to wrap it with an element(`<a> or <button>`) that is clickable because we need to click on it in order to show the menu links that may be hidden in the mobile version of the web page 👍